IPython/core/extensions.py

```diff
@@ -13,7 +13,11 @@
 from traitlets import Instance
 from IPython.utils.py3compat import PY3
 if PY3:
-    from imp import reload
+    try: 
```

Comment:
scratch that. imp.reload is aliased to the build-in in py2, removed, and drop the PY3 import.